tests/upgrade/test_upgrade.py

```
@@ -0,0 +1,118 @@
+import time
+import sys
+import os
+import threading
+import docker
+import docker.models
+import docker.models.containers
+
+sys.path.append(os.path.dirname(os.path.abspath(__file__)) + "/../flow/")
+
+from common import *
```

Ruff

`from common import *` used; unable to detect undefined names

(F403)